pkg/services/sqlstore/dashboard.go

```diff
@@ -68,17 +69,43 @@ func SaveDashboard(cmd *m.SaveDashboardCommand) error {
 			}
 		}
 
-		affectedRows := int64(0)
+		parentVersion := dash.Version
+		version, err := getMaxVersion(sess, dash.Id)
```

Comment:
Yeah you're right, this is unnecessary now. I think at point during dev on this I handled the version incrementing differently which lead to cases where we needed something like this, but had since changed the logic so that it wasn't necessary anymore.